libs/designsystem/src/lib/components/chart/options/column.ts

```diff
@@ -0,0 +1,121 @@
+import { Options } from 'highcharts';
+
+import { DesignTokenHelper } from '../../../helpers/design-token-helper';
+
+const getColor = DesignTokenHelper.getColor;
+const fontSize = DesignTokenHelper.fontSize;
+
+function colorPoints() {
+  return function() {
+    var series = this.series;
+    for (var i = 0, ie = series.length; i < ie; ++i) {
+      var points = series[i].data;
+      for (var j = 0, je = points.length; j < je; ++j) {
+        if (points[j].graphic) {
+          points[j].graphic.element.style.stroke = getColor('secondary').value;
+        }
+      }
+    }
+  };
+}
+
+export const getColumnOptions = (data: number[], categories: string[]) =>
```

Comment:
I think this is better than injecting as we can pass params

Reply:
Not quite following, let's talk...